src/edit.js

```diff
@@ -1,5 +1,21 @@
-const EditView = () => {
-	return <div>The front end is alive.</div>;
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon } from '@wordpress/components';
+
+const EditView = ( { className } ) => {
+	return (
+		<div className={ className }>
```

Comment:
There's a `Placeholder` component you could use for this: https://github.com/WordPress/gutenberg/tree/master/packages/components/src/placeholder

Reply:
For some reason, the icon that comes out of the icon package and the `dashicon` that is provided to the block config are different. I definitely want to keep the icon because when it's not focused, it's not evident that the block is related. 

It's also a bit confusing to me that the `@wordpress/icons` package doesn't resolve unless its explicitly installed. Any idea what's going on there?

![ScreenShot](https://d.pr/i/Uep05z.png)

Reply:
I just tried out `<Placeholder icon="calendar" label="Placeholder">` enclosing the text, and got matching icons:

<img width="587" alt="Screen Shot 2020-03-06 at 3 58 04 pm" src="https://user-images.githubusercontent.com/7200686/76054026-5b1b5200-5fc3-11ea-9c07-3140f1f0cd58.png">

I have no idea why, just blindly copying code.

Reply:
Weird. The example `https://github.com/WordPress/gutenberg/tree/master/packages/components/src/placeholder` has the SVG exported and then the reference passed in. I guess, if you just pass the string it falls back to dashicons. That's unexpected. I can update.

Reply:
Yeah, that's apparently how it works:

`https://github.com/WordPress/gutenberg/blob/8a94254b6adac030f5a6ffd8e718972e1f19f48e/packages/components/src/icon/index.js#L21`

Reply:
i think they've been updating to move away from dashicons in general, so `@wordpress/icons` is a new package of new icons, but it's still in transition — see https://github.com/WordPress/gutenberg/pull/20464

> It's also a bit confusing to me that the @wordpress/icons package doesn't resolve unless its explicitly installed. 

Might be a versioning issue, if you're not using the GB plugin, or using an older version.